src/build/build.ts

```diff
@@ -12,48 +12,38 @@
  * http://polymer.github.io/PATENTS.txt
  */
 
-import * as del from 'del';
 import * as path from 'path';
 import * as logging from 'plylog';
 import {dest} from 'vinyl-fs';
 import mergeStream = require('merge-stream');
 import {PolymerProject, addServiceWorker, SWConfig} from 'polymer-build';
 
 import {OptimizeOptions, getOptimizeStreams} from './optimize-streams';
-import {ProjectConfig} from 'polymer-project-config';
+import {ProjectConfig, ProjectBuildOptions} from 'polymer-project-config';
 import {PrefetchTransform} from './prefetch';
 import {waitFor, pipeStreams} from './streams';
 import {loadServiceWorkerConfig} from './load-config';
 
 const logger = logging.getLogger('cli.build.build');
 
-const buildDirectory = 'build/';
-
-export interface BuildOptions extends OptimizeOptions {
-  addServiceWorker?: boolean;
-  swPrecacheConfig?: string;
-  insertPrefetchLinks?: boolean;
-  bundle?: boolean;
-};
-
 export async function build(
-    options: BuildOptions, config: ProjectConfig): Promise<void> {
+    options: ProjectBuildOptions, config: ProjectConfig): Promise<void> {
```

Comment:
That's my question: why doesn't it care about `config.builds`? Coming into this code, if I know that ProjectConfig has a builds property, I wonder why this doesn't use it. I think you're saying this only performs one of the builds contained in there, which would answer my question, but I think this could use some comments to that effect if so.